runelite-client/src/main/java/net/runelite/client/plugins/lizardmenshaman/LizardmenShamanPlugin.java

```diff
@@ -102,12 +102,9 @@ protected void shutDown() throws Exception
 	@Subscribe
 	public void onChatMessage(ChatMessage event)
 	{
-		if (this.notifyOnSpawn)
+		if (this.notifyOnSpawn && event.getMessage().contains(MESSAGE))
```

Comment:
I think checking messagetype as well may be a bit faster, if you do that before checking the message

Reply:
Marking as reminder
- Don't know the message type

Reply:
chatMessage.getType() == ChatMessageType.GAMEMESSAGE)